admin_guide/sdn_troubleshooting.adoc

```diff
@@ -495,6 +495,54 @@ normal operation, so do not be surprised to see entries there.
 
 Check external firewalls, if any, allow traffic to the target address (this is site-dependent, and beyond the purview of this guide).
 
+== Debugging Virtual Networking
+
+=== Builds on a Virtual Network are Failing
+
+If you are installing OpenShift using a virtual network (for example,
+OpenStack), and a build is failing, the node maximum transmission unit (MTU)
+might not be compatible with the network MTU (for example, eth0).
+
+For a build to complete successfully, the MTU of an SDN must be less than the
+eth0 network MTU in order to pass data to between node hosts.
+
+. Check the MTU of your network by running the `ip addr` command:
++
+====
+----
+# ip addr
+---
+2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP qlen 1000
+    link/ether fa:16:3e:56:4c:11 brd ff:ff:ff:ff:ff:ff
+    inet 172.16.0.0/24 brd 172.16.0.0 scope global dynamic eth0
+       valid_lft 168sec preferred_lft 168sec
+    inet6 fe80::f816:3eff:fe56:4c11/64 scope link 
+       valid_lft forever preferred_lft forever
+---
+----
+====
++
+The MTU of the above network is 1500.
+
+. The MTU in your node configuration must be lower than the network value. Check
+the `mtu` in the node configuration of the targeted node host:
```

Comment:
Use parameter markup ([bold-code](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference)) for `mtu` throughout when referring to the parameter name (but not when referring generally to MTU in all-caps).